packages/cashier/src/components/crypto-transactions-history/__tests__/crypto-transactions-renderer.spec.tsx

```diff
@@ -34,11 +36,14 @@ describe('<CryptoTransactionsRenderer />', () => {
             amount: 0.005,
             id: '3',
             is_valid_to_cancel: 1,
-            status_code: 'LOCKED',
+            status_code: 'LOCKED' as TStatusCode,
             status_message:
                 "We're reviewing your withdrawal request. You may still cancel this transaction if you wish. Once we start processing, you won't be able to cancel.",
             submit_date: 1640603927,
-            transaction_type: 'withdrawal',
+            transaction_hash: '',
```

Comment:
Tests fail somehow if this property is truthy 🤔 
The `should show the proper data in Desktop mode` test in particular